src/components/index.js

```diff
@@ -0,0 +1,3 @@
+import Button from './Button/Button';
+
+export default { Button };
```

Comment:
add https://www.npmjs.com/package/babel-plugin-transform-export-default
which will let you do `export {Button} from './Button/Button`